plugins/tap.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: tap
+spec:
+  version: v0.1.1
+  homepage: https://soluble-ai.github.io/kubetap/
+  shortDescription: "interactively proxy Kubernetes Services with ease"
```

Comment:
- please capitalize
- "with ease" is subjective, consider removing

Reply:
I've added caps. The "with ease" is part of the [project's tagline]( https://github.com/soluble-ai/kubetap) and I have not removed it.